### PR TITLE
#2092 swagger doc: fix for runs + readme description enhanced

### PR DESCRIPTION
**Pull request by dk1844**
Swagger doc fix:
 - runs V3 were missing from the swagger UI
 - Brief API description in README has been updated.

#2092 

## Release notes suggestion
 - Swagger doc for API V3 Runs added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,12 +49,17 @@ The project is comprised of four main components:
 
 ### REST API
 The REST API exposes the Enceladus endpoints for creating, reading, updating and deleting the models, as well as other functionalities.
-The main three models used are:
+The main models used are:
  - **Dataset**: Specifies where the dataset will be read from on HDFS (**RAW**), the conformance rules that will be applied to it, and where it will land on HDFS once it is conformed (**PUBLISH**)
  - **Schema**: Specifies the schema towards which the dataset will be standardized
  - **Mapping Table**: Specifies where tables with master reference data can be found (parquet on HDFS), which are used when applying Mapping conformance rules (e.g. the dataset uses **Germany**, which maps to the master reference **DE** in the mapping table)
+ - **Property Definitions**: Datasets may be accompained by properties, but these are not free-form - they are bound by system-wide property definitions.
+ - **Runs**: Although not able to be defined by users, Runs provide important overview of Standardization & Conformance jobs that have been carried out.
```
Comment (benedeki) [inline README.md:57]: **Minor:** 
```suggestion
 - **Runs**: Although not able to be defined by users, Runs provide important overview of _Standardization_ & _Conformance_ jobs that have been carried out.
```

File: README.md [hunk 2]
```diff
@@ -49,12 +49,17 @@ The project is comprised of four main components:
 
 ### REST API
 The REST API exposes the Enceladus endpoints for creating, reading, updating and deleting the models, as well as other functionalities.
-The main three models used are:
+The main models used are:
  - **Dataset**: Specifies where the dataset will be read from on HDFS (**RAW**), the conformance rules that will be applied to it, and where it will land on HDFS once it is conformed (**PUBLISH**)
  - **Schema**: Specifies the schema towards which the dataset will be standardized
  - **Mapping Table**: Specifies where tables with master reference data can be found (parquet on HDFS), which are used when applying Mapping conformance rules (e.g. the dataset uses **Germany**, which maps to the master reference **DE** in the mapping table)
+ - **Property Definitions**: Datasets may be accompained by properties, but these are not free-form - they are bound by system-wide property definitions.
```
Comment (benedeki) [inline README.md:56]: **Minor for discussion:** Maybe I would order the sections as the Menas buttons?

**Review by benedeki — APPROVED** [review]
- [x] code reviewed
- [ ] pulled
- [ ] built
- [ ] run

File: README.md (hunk 2 above)
Comment (dk1844) [reply README.md]: Makes sense, done, thanks.

**Review by Zejnilovic — APPROVED** [review]

**Review by benedeki — APPROVED** [review]
- [X] code reviewed
- [X] pulled
- [X] built
- [x] run

**Comment by benedeki** [comment]
Run and checked the Swagger documentation - the Runs are there now.

**Pull request merged**